docs/postman/AAS-Service.postman_collection.json

```diff
@@ -0,0 +1,1127 @@
+{
```

Comment:
Why is this a new separate Postman Collection file?

Reply:
Good point... Seems that was exported with a different file name
I pushed an update